### DISCUSSION TITLE
blog/2017/03/19/amortization-and-persistence-via-lazy-evaluation

### DISCUSSION THREAD

**Discussion by giscus** [kunigami/kunigami.github.io#10] [Announcements]
# blog/2017/03/19/amortization-and-persistence-via-lazy-evaluation

NP-Incompleteness: 

https://www.kuniga.me/blog/2017/03/19/amortization-and-persistence-via-lazy-evaluation.html

**Comment by nothankyouzzz** [comment]
helps a lot! thanks for your summary.
